internal/runners/packages/import.go

```diff
@@ -48,25 +49,34 @@ func NewImportRunParams() *ImportRunParams {
 
 // Import manages the importing execution context.
 type Import struct {
-	out output.Outputer
+	out  output.Outputer
+	proj *project.Project
 	prompt.Prompter
 }
 
 type primeable interface {
 	primer.Outputer
 	primer.Prompter
+	primer.Projecter
 }
 
 // NewImport prepares an importation execution context for use.
 func NewImport(prime primeable) *Import {
 	return &Import{
 		prime.Output(),
+		prime.Project(),
 		prime.Prompt(),
 	}
 }
 
 // Run executes the import behavior.
 func (i *Import) Run(params ImportRunParams) error {
+	err := i.run(params)
+	headless.Notify(i.out, i.proj, err)
```

Comment:
missing cmdNames

Reply:
Theses are supposed to be missing the cmdNames; They should only print a general message, and not a command-specific message.

Reply:
Where do you get that from? Looking at the AC's again I don't see these commands in there at all.

Reply:
hmm. I might have misunderstood. Pete had stated that all commands run headless should get a generic message, but some should get a more specific message. I'll review the AC and what Pete said in Slack and update the AC, then ping you for approval.

Reply:
Please do not update ACs without my say so. Slack does not trump ACs for stories that are in our sprint commit.